src/store/reducers/executeQuery.ts

```diff
@@ -233,6 +233,7 @@ export const executeQueryApi = api.injectEndpoints({
                                 querySettings.tracingLevel && enableTracingLevel
                                     ? TracingLevelNumber[querySettings.tracingLevel]
                                     : undefined,
+                            limit_rows: querySettings.limitRows || undefined,
```

Comment:
will it be possible to run query without this setting at all?

Reply:
now it is running without this setting

or I didnt get your idea

Reply:
Just wanted to double-check that `limit_rows` may be `undefined` and nothing breaks!